packages/components/src/Avatar/_docs/Avatar.mdx

```diff
@@ -15,7 +15,7 @@ import * as AvatarStories from "./Avatar.stories"
 
 
 <Installation
-  installCommand="yarn add @kaizen/components"
+  installCommand="pnpm add @kaizen/components"
```

Comment:
I wonder about this, I think technically the majority of consuming apps are still `yarn`

Reply:
Also we could probably bake the install command into the component since its the same for them all? Maybe even the import statement could be partially templated

Reply:
Yeah I was actually thinking that haha. Maybe it can just list both 🤷🏻‍♀️

Reply:
Might be too thick...

Reply:
`(yarn|pnpm) add @kaizen/components`? 😂 Will see